yamlprivateh.go

```diff
@@ -104,12 +104,12 @@ func as_hex(b []byte, i int) int {
 }
 
 // Check if the character is ASCII.
-func is_ascii(b []byte, i int) bool {
+func isAscii(b []byte, i int) bool {
```

Comment:
Maybe this to respect the acronym

```suggestion
func isASCII(b []byte, i int) bool {
```